supporting_content/dimensions/commitmentdiscounttype.md

```diff
@@ -0,0 +1,55 @@
+# Column: CommitmentDiscountType
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider | Data set                 | Column                    |  Example Values  |
+|----------|--------------------------|---------------------------|------------------|
+| AWS | CUR                      | Not available   | N/A |
```

Comment:
For AWS can be calculated: if reservation/ReservationARN <> '' then Usage, else if savingsPlan/SavingsPlanArn
<> '' then Spend, else null

Reply:
Ok, added to text